core/src/main/java/nl/nn/adapterframework/core/PipeForwards.java

```diff
@@ -0,0 +1,34 @@
+/*
+   Copyright 2022 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package nl.nn.adapterframework.core;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import lombok.Getter;
+
+public class PipeForwards {
```

Comment:
voorheen heette dit global-forwards is het verstanding om dit nu forwards te noemen?

Reply:
element heet nu weer global-forwards

Reply:
kijkt hij puur naar de setter en niet de classname?